macdmp.c

```diff
@@ -412,7 +421,7 @@ main (int argc, char **argv)
   output_word_format = &its_word_format;
   verbose = 0;
 
-  while ((opt = getopt (argc, argv, "vc:t:x:W:")) != -1)
+  while ((opt = getopt (argc, argv, "vc:t:x:W:N:")) != -1)
```

Comment:
Worth adding these other options to usage()? It only mentions x/t/c at the moment.

Reply:
Thanks, good point!